app.js

```diff
@@ -139,6 +139,7 @@ function redirectOldUrl(req, res, next) {
   }
   if (req.originalUrl.startsWith('/apis/')) {
     res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Access-Control-Allow-Credentials', 'true');
```

Comment:
I am guessing this is to get the cookie to work when fetching the endpoints from sdg-inno-commons. Does this pose any potential security issues?

Reply:
Yes, this is towards sharing of cookies when calling the apis. No, there is no security issue because connection and cookies are encrypted and server based.

Reply:
ok great. thanks